.github/workflows/release.yml

```diff
@@ -39,10 +39,21 @@ jobs:
           printf "appimage_tag=${APPIMAGE_TAG}\n" >> $GITHUB_OUTPUT
 
   linux:
-    runs-on: ubuntu-20.04
     needs: setup
+    strategy:
+      fail-fast: false
+      matrix:
+        runner: [ ubuntu-20.04, ubuntu-24.04-arm ]
+        include:
+          - runner: ubuntu-20.04
+            arch: x86_64
+            cc: gcc-10
+          - runner: ubuntu-24.04-arm
+            arch: aarch64
+    runs-on: ${{ matrix.runner }}
     env:
-      CC: gcc-10
+      CC: ${{ matrix.cc }}
+      LDAI_NO_APPSTREAM: 1 # skip checking (broken) AppStream metadata for issues
```

Comment:
Note: This is necessary to make the current `linuxdeploy` (only on Ubuntu-24.04) not bail out on our broken AppStream metadata.

Reply:
Why not fix our AppStream metadata?